geode-pulse/src/main/java/org/apache/geode/tools/pulse/internal/PulseAppListener.java

```diff
@@ -88,11 +92,40 @@ public void contextInitialized(ServletContextEvent event) {
       repository.setPort(System.getProperty(PulseConstants.SYSTEM_PROPERTY_PULSE_PORT,
           PulseConstants.GEMFIRE_DEFAULT_PORT));
 
-      // SSL, all the other system properties are already set in the embedded VM
       repository.setUseSSLManager(
           Boolean.valueOf(System.getProperty(PulseConstants.SYSTEM_PROPERTY_PULSE_USESSL_MANAGER)));
       repository.setUseSSLLocator(
           Boolean.valueOf(System.getProperty(PulseConstants.SYSTEM_PROPERTY_PULSE_USESSL_LOCATOR)));
+
+      Object sslConfigObject =
```

Comment:
If you pass the SSLConfig object to here, you can just directly set the entire SSLConfig object into Repository object